recipe/meta.yaml

```diff
@@ -17,18 +17,21 @@ requirements:
     - setuptools
     - cython >=0.23
     - python
-    - six >=1.4
+    - six >=1.7.3
     - numpy x.x
-    - scipy >=0.9
+    - scipy >=0.17
+    - numpydoc >=0.6
   run:
     - python
-    - six >=1.4
-    - numpy x.x
```

Comment:
I don't think this line should be removed? We need this in `build` and `run` as we are linking to NumPy (unless that has changed). Please re-render after addressing this and it should fix the NumPy issue in the CIs.